plasma_framework/test/src/exits/payment/PaymentStartInFlightExit.test.js

```diff
@@ -570,5 +575,239 @@ contract('PaymentInFlightExitRouter', ([_, alice, bob, carol]) => {
                 );
             });
         });
+
+        describe('canonicity challenge', () => {
```

Comment:
Can we move the test to a new file? eg. `PaymentChallengeIFENotCanonical.test.js`
After previous refactor (router-controller) we've move tests to reflect each separate function also to avoid a super huge test file.